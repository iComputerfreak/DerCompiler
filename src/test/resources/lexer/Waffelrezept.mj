500 g 	Zucker
500 g 	Margarine, zimmerwarme
10 	Ei(er), verquirlte
2 Pck. 	Vanillinzucker
1 Schuss 	Rum
1 kg 	Mehl
1 Liter 	Milch
1 Pck. 	Backpulver
