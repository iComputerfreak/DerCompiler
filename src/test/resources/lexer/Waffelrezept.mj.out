integer literal 500
identifier g
identifier Zucker
integer literal 500
identifier g
identifier Margarine
,
identifier zimmerwarme
integer literal 10
identifier Ei
(
identifier er
)
,
identifier verquirlte
integer literal 2
identifier Pck
.
identifier Vanillinzucker
integer literal 1
identifier Schuss
identifier Rum
integer literal 1
identifier kg
identifier Mehl
integer literal 1
identifier Liter
identifier Milch
integer literal 1
identifier Pck
.
identifier Backpulver
EOF
